### ISSUE TITLE
CVE-2023-4863 impacting libwebp 1.0.2

### ISSUE THREAD

**Issue by delroth** [chai2010/webp#61]
Hi!

This Go library vendors libwebp 1.0.2, which is vulnerable to CVE-2023-4863 (critical severity buffer overflow in libwebp image decoding). Upstream has a 1.0.3 available with the vulnerability fixed: https://github.com/webmproject/libwebp/tree/1.0.3

Could you please update the vendored libwebp and tag a new release of this library so dependents can get updated?

Thank you!

**Comment by LascarPaul** [comment]
@delroth did you mean "Upstream has a **1.3.2** available with the vulnerability fixed"

**Comment by delroth** [comment]
No, I meant 1.0.3 - it isn't actually officially released, but there is an upstream Git branch with the patch backported on top of 1.0.x. I even linked to it.

Updating to 1.3.2 would obviously be better, but for this vuln in particular there's a trivial update path which shouldn't involve any API backwards compatibility problems.

**Comment by nikooo777** [comment]
please @chai2010 can you push this change?

**Comment by trunov** [comment]
@chai2010 hey, please update the package.

**Comment by nikooo777** [comment]
nice, that happened! this can finally be closed now

**Comment by cogtea** [comment]
Thanks for the update, Any idea when release 1.0.3 will be available?
